lib/value/Cargo.toml

```diff
@@ -29,7 +29,7 @@ lua = ["mlua"]
 api = ["async-graphql"]
 json = ["serde", "serde_json"]
 test = []
-arbitrary = ["quickcheck"]
+arbitrary = ["quickcheck", "test"]
```

Comment:
This isn't needed once the arbitrary changes are reverted

Reply:
There's no `From<f64>` without the `test` feature, which was the original failure. I'll get rid of this and adjust the `Arbitrary` implementation to use `NotNan` directly.